### PR TITLE
[mungegithub] ApprovalNotification Add Json

### PR DESCRIPTION
**Pull request by grodrigues3**
- approval notification writes json for gubernator to read and display on pr dashboard

### CODE REVIEW

**Comment by grodrigues3** [comment]
@rmmh does this do what you want more or less 

(i'll dry run it on my local repos before merging)

File: mungegithub/mungers/approval-handler.go [hunk 1]
```diff
@@ -363,6 +364,15 @@ func (h *ApprovalHandler) getMessage(obj *github.MungeObject, ownersMap map[stri
 	if isFullyApproved {
 		title = "This PR is **APPROVED**"
 	}
+	approverList := suggestedApprovers.List()
+	forMachine := make(map[string][]string)
```
Comment (rmmh) [inline mungegithub/mungers/approval-handler.go]: this would be shorter as `forMachine := map[string][]string{"approvers": suggestedApprovers.List()}

**Comment by apelisse** [comment]
Fixes #1727

**Comment by grodrigues3** [comment]
fixes the boilerplate modification from before too

**Comment by apelisse** [comment]
Awesome!

**Pull request merged**